src/tests/ftest/dfuse/read.py

```diff
@@ -23,7 +23,9 @@ def test_dfuse_pre_read(self):
         Read one large file entirely using pre-read.  Read a second smaller file to ensure that
         the first file leaves the flag enabled.
 
-        :avocado: tags=all,full_regression
+        TODO: Check this with 128k I/O size and non-128K I/O size.
+
+        :avocado: tags=all,daily_regression
```

Comment:
Just checking that moving this test to daily from weekly is intentional.

Reply:
Parts of this PR, including this bit come from https://github.com/daos-stack/daos/pull/15298

I put this is here as a placeholder to myself, I didn't know full_regression means weekly, I assumed from the name it was less frequent than that and this is here as a reminder that I need to verify the frequency we're running this test.